app/assets/stylesheets/modules/_course.styl

```diff
@@ -159,7 +159,7 @@ rating(color)
     z-index 1
 
   .course-details
-    width 50%
+    width 80%
```

Comment:
Making this 90%, along with the max-width, is a little better I think.

Reply:
Sorry for the discrepency,I'll change that